packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -1,21 +1,37 @@
 const compatibility = require('./compatibility');
 const { createDelegatee, delegatingVisitor } = require('./delegation');
 const features = require('./features');
+const polyfilling = require('./polyfilling');
 const targetRuntimes = require('./targetRuntimes');
 
 const targets = targetRuntimes();
-const delegateeConfigs = compatibility
-  .forbiddenFeatures(features, targets)
-  .map((feature) => feature.ruleConfig);
+const unsupportedFeatures = compatibility.unsupportedFeatures(features, targets);
 
 module.exports = {
   meta: {
     type: 'problem',
-    schema: [], // no options
+    schema: [
+      {
+        'type': 'object',
+        'properties': {
+          'polyfills': {
+            'type': 'array',
+            'items': {
+              'type': 'string'
+            }
+          }
+        },
+        'additionalProperties': 'false',
+      }
+    ]
   },
   create(context) {
-    const visitors = delegateeConfigs.map((config) => createDelegatee(config, context));
+      const polyfills = context.options?.[0]?.polyfills;
```

Comment:
Would be good to normalise this (`?? []`) here at the point it comes into our code, instead of needing `?.` in `isPolyfilled`